lib/configs/flat/recommended.js

```diff
@@ -17,21 +17,21 @@ module.exports = {
   },
   plugins: {
     prettierPlugin,
-    eslintComments,
+    'eslint-comments': eslintComments,
```

Comment:
Just curious. Why is it necessary to revert to the original ones?

Reply:
This way we can use the same names in dotcom. Or else we'd have to update the names in the monolith to `eslintComments`.

Reply:
Thanks for clarifying!